config/sys.config

```
@@ -38,8 +38,17 @@
                 162, 59, 180, 164, 195, 18, 72, 169, 72, 153, 183, 114, 159, 34>>},
 
         {listen_addresses, ["/ip4/0.0.0.0/tcp/44158"]},
-        {store_implicit_burns, false},
-        {store_htlc_receipts, false},
+        {store_htlc_receipts, true},
+        {store_implicit_burns, true},
+        {store_historic_balances, true},
```

```suggestion
        {store_htlc_receipts, false},
        {store_implicit_burns, false},
        {store_historic_balances, false},
```

Let's keep these false and just on for the rosetta config. I forget exactly why we turned them off but needed to for another node deployment. 

cc: @jadeallenx @madninja @evanmcc